data/analytics/events.yml

```diff
@@ -1058,11 +1058,24 @@
 - name: see all link
   events:
     - name: see all link click
```

Comment:
There's also a 'link' version of this event, when 'See all updates' goes to another page: https://www.gov.uk/hmrc-internal-manuals/air-passenger-duty

So that will need to be added. Feel free to do it in a separate PR (or get me to do it)

Reply:
Thanks @AshGDS, great detail. Have updated the `Update see all updates GA4 event data` commit with this addition.